packages/maestro/.eslintrc

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Isn't there a shared eslint config for all the packages in the monorepo?

Reply:
Yep, there is, but the `typesript` 

- Doesn't respect prettier config
- Has a bit of outdated configuration (wrong project resolution )
- Isn't strict enough 

We talked with @justintemps about config refresh because it's 2 years old. If we like the new ruleset, we can make it a new default.

Reply:
@GGKapanadze and @bashlk I'm ok with this, we can leave it here for now and then use it as a starting point when we tackle #1045